README.md

```diff
@@ -15,7 +15,8 @@ as [Endpoints](https://hilla.dev/docs/lit/guides/endpoints),
 [Reactive Endpoints](https://hilla.dev/docs/lit/guides/reactive-endpoints)
 and [Security](https://hilla.dev/docs/lit/guides/security).
 
-**NOTE**: This is an **unofficial community extension**, and it is **not** directly
+**NOTE**: This is an **unofficial community extension**, and it is **not**
+directly
```

Comment:
Why the line break?

Reply:
Probably auto formatted by the IDE, I'll fix it

Reply:
done